### ISSUE TITLE
[ Enhancement ] April Fool's Office Hours Stack

### ISSUE THREAD

**Issue by danlliu** [CarsonHoffman/office-hours-queue#28]
On April 1, courses should have an option to convert their office hours queue into an office hours stack! Just for fun :)

**Comment by CarsonHoffman** [comment]
Do you have support from your course on this? (i.e. will anybody actually use it if I implement it? 😛)

**Comment by danlliu** [comment]
Yup! 281 Discord (not officially "my" course but I've definitely seen this sentiment in quite a few places):

<img width="545" alt="Screen Shot 2022-03-19 at 11 36 46" src="https://user-images.githubusercontent.com/66093915/159127721-5c31f4a5-9aa2-4533-b710-0da226671b0c.png">

<img width="1106" alt="Screen Shot 2022-03-19 at 11 37 21" src="https://user-images.githubusercontent.com/66093915/159127744-85975422-16ca-4f49-8d0c-5fecda5d21ab.png">

**Comment by seshrs** [comment]
While I'm fully for an April Fools prank, I'd caution against changing the functionality of the queue, just because it's course infra and plays a part in student grades!

That said, if it's just a name-change on April 1, I don't think you need to make it opt-in. (Or we can brainstorm other ideas here 😃 )

**Comment by danlliu** [comment]
One of my ideas is that we could add a setting that allows queues to be FIFO or LIFO, and then on April Fools, courses could add a new LIFO office hours option, while keeping their old ones for genuine student questions.
